src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import React from 'react';
+import { Tab } from '../../Types/types';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+}
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  const tabClickHandler = (tab: Tab) => {
```

Comment:
This function isn't handler. 

```suggestion
  const selectTab= (tab: Tab) => {
```